### PR TITLE
League of Legends Weekly Champion Rotation

### PR DESCRIPTION
**Pull request by dpaiv0**
(i know the branch is "league-**daily**-champ", no blame)

+ Supports low-level champion rotation (champions that are free to players under level 10) through a flag (--newplayers or --np)
command being executed without the --np flag
command being executed with the --np flag

### CODE REVIEW

**Comment by pedrofracassi** [comment]
Can we have a canvas-generated image? Kappa

**Comment by dpaiv0** [comment]
WAIT

ARE YOU THINKING THE SAME AS I AM?

**Comment by dpaiv0** [comment]
mano eu não sei canvas alguem que faz ai plmds

File: src/apis/LeagueOfLegends.js [hunk 1]
```diff
@@ -53,14 +52,12 @@ module.exports = class LeagueOfLegends extends APIWrapper {
     })
   }
 
-  request (endpoint) {
-    return fetch(API_URL + endpoint)
-      .then(res => res.json())
+  fetchChampionRotation () {
+    return this.request(`/platform/v3/champion-rotations?api_key=${process.env.RIOT_API_KEY}`, true).then(u => u)
```
Comment (fernando457829) [inline src/apis/LeagueOfLegends.js]: wtf? `.then(u => u)`?

File: src/apis/LeagueOfLegends.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
-const API_URL = 'https://ddragon.leagueoflegends.com'
+const DATADRAGON_URL = 'https://ddragon.leagueoflegends.com'
+const RIOT_API_URL = 'https://na1.api.riotgames.com/lol'
 
 module.exports = class LeagueOfLegends extends APIWrapper {
   constructor () {
     super({
-      name: 'lol'
+      name: 'lol',
+      envVars: [ 'RIOT_API_KEY' ]
```
Comment (fernando457829) [inline src/apis/LeagueOfLegends.js]: This env var isn't on `.env.example` file

**Comment by dpaiv0** [comment]
oh yeah wait

**Pull request merged**